### ISSUE TITLE
only support Kafka 1.0 above

### ISSUE THREAD

**Issue by robbinhan** [birdayz/kaf#111]
I used kafka 0.10.1.1 before and found that it has been unavailable. Than debug found that it only supports kafka 1.0 and above.

For a new tool, I think it is understandable that the old version is no longer supported, but I hope I can explain it in the readme

**Comment by Spbau** [comment]
bad news

**Comment by birdayz** [comment]
this is actually not true. some features - IIRC mostly group related - require a certain broker version, i think 1.0.0. kaf allows you to set the version in the config manually - i agree that it's neither documented nor obvious.

**Comment by birdayz** [comment]
i added the --broker-version flag to add-cluster. also, i updated to examples/basic.yaml file to show how it can be edited directly in the config file.

Example:
```
Flags:
      --broker-version string   Broker Version. Available Versions: [0.8.2.0 0.8.2.1 0.8.2.2 0.9.0.0 0.9.0.1 0.10.0.0 0.10.0.1 0.10.1.0 0.10.1.1 0.10.2.0 0.10.2.1 0.11.0.0 0.11.0.1 0.11.0.2 1.0.0 1.1.0 1.1.1 2.0.0 2.0.1 2.1.0 2.2.0 2.3.0 2.4.0]
```

to come back with the initial remark in this issue: it's possible to use older versions, i think some commands in `kaf group` will not work, but most of the rest should. didn't test it though.

**Comment by robbinhan** [comment]
I added `version` in the configuration, tested the` 0.10.1.1` version, and found that `kaf group` is possible, and `kaf topics` will prompt that the version is not supported

Finally, I still hope to explain in the document

thx
